Library/Homebrew/cmd/cleanup.rb

```diff
@@ -19,14 +19,9 @@ def cleanup
 
   def cleanup_logs
     return unless HOMEBREW_LOGS.directory?
-    prune = ARGV.value "prune"
-    if prune
-      time = Time.now - 60 * 60 * 24 * prune.to_i
-    else
-      time = Time.now - 60 * 60 * 24 * 7 * 2 # two weeks
-    end
+    prune = prune_flag 14 # two weeks unless overridden by argument
```

Comment:
The reason for a separate method is that it is referenced in two different places and I wanted to avoid duplicating the logic. I like the suggested name change, though; will push it in a bit.